### ISSUE TITLE
d0 not defined when not using control file

### ISSUE THREAD

**Issue by MBrancoLab** [FredHutch/SEACR#20]
I haven't really tried to figure out what d0 is, but this variable is not defined when a percentile threshold is used (without control set), which causes an issue at one of the write.table.

**Comment by mpmeers** [comment]
Hi,

There are a couple of very obvious issues here that I missed. First, I forgot to update the call from to SEACR_1.1.R to SEACR_1.2.R in SEACR_1.2.sh, so I've fixed that. Your issue, however, I think is related to the fact that in v1.2, the write.table takes a d0 variable that I created as a new filter for sparse signal block composition where SEACR was failing with very low quality datasets. However, I did not assign a d0 to the percentile threshold part of the loop, so it fails to write the whole thing. I've now added an arbitrary d0 to that part of the loop so that it goes through properly.

I created a new tag (SEACRv1.2) assigned to these changes, and they all exist on the master branch. Let me know if this fixes the issue, and apologies for not catching those simple bugs.

Best,

Mike
